paper.tex

```diff
@@ -68,28 +68,62 @@ \section*{Ancestral graphs and stochastic processes}
 % See https://github.com/tskit-dev/what-is-an-arg-paper/discussions/13
 % for discussions on this section
 
-The coalescent
-process~\citep{kingman1982coalescent,kingman1982genealogy,hudson1983testing,
-tajima1983evolutionary}
-
-The Ancestral Recombination Graph (ARG) was introduced by
-Griffiths~\citep{griffiths1991two,griffiths1997ancestral}.
-The ``big'' ARG~\citep{ethier1990two},
-and the ``little ARG'' traversed by
-Hudson's algorithm~\citep{hudson1983properties}.
-
+The simplest prominent example of genetic ancestry generated by a stochastic process
+is the coalescent process \citep{kingman1982coalescent,kingman1982genealogy,
+hudson1983testing, tajima1983evolutionary}, which describes the random
+ancestral tree at one non-recombining locus obtained by sampling a relatively
+small number of individuals at random from a large population.
+
+In a recombining organism, the genetic ancestry of a longer segment of DNA is 
+no longer described by a single tree. Instead, ancestral relationships in a
+sample can be represented by a sequence of correlated, marginal trees along the
+genome. The Ancestral Recombination Graph (ARG) was introduced by
+Griffiths~\citep{griffiths1991two,griffiths1997ancestral} as a branching-coalescing 
+stochastic process whose realisations are graphs which into which all of the
+marginal ancestral trees are embedded. Coalescences encode common ancestry
+as they do in the coalescent, while branching events encode points at which a
+marginal tree changes from one position along the genome to the next.
+
+There are several formulations of the ARG, which vary by
```

Comment:
```suggestion
% jk-note: this isn't quite right I think, as it's muddling the distinction between a graph stochastic process 
% definition with the graph as the encoding of the marginal trees. 
There are several formulations of the ARG, which vary by
```
I don't think this is quite right, but I can't see how to tweak it immediately. I suggest we put in the comment and look at it again when we have more text to connect with.

Reply:
I agree. Good spot @jeromekelleher

Reply:
Just changing the verb from "capture" to "generate", or something to that effect. Either way, I agree this needs clarifying.